.github/workflows/build.yml

```diff
@@ -10,21 +10,23 @@ env:
   HUB: ghcr.io/${{ github.repository }}
 
 jobs:
-  build:
-    runs-on: self-hosted
+  build_proxyv2:
+    runs-on: ubuntu-20.04
```

Comment:
Is this version in sync with upstream?

Reply:
Yes. The upstream uses Ubuntu 20.04 as build container.

https://github.com/intel/envoy/blob/0ef5e14bc06413fb37f608e80b140767ad371c26/.azure-pipelines/pipelines.yml#L216